### ISSUE TITLE
Feedback: Overview

### ISSUE THREAD

**Issue by cabergh** [whitead/dmol-book#79]
Thanks for a great book that I've really enjoyed reading! The format with clear examples accompanied with executable code blocks and the clear focus on key concepts made grasping the most important concepts easy. I learned many new things and discovered new angles of looking at previously learned material. I didn't make it through the whole book, but I have some feedback for the first 9 chapters, which I have divided up into separate issues. 

- Overall, I think the overview could be better formulated. This is probably the part most students will read before they decide if the book is worth their time! I think the authors could “sell” deep learning a bit more in the beginning to grab the students’ interest and then end with “although powerful, it’s not a new paradigm of science or replacement for a chemist...”. Some more specific comments:

- The first paragraph could be better formulated. E.g. the second sentence could be clearer by making a distinction between the trained function and mathematical (ground truth) function, e.g. as: 
“Specifically, the purpose is to connect some input data (features) and output data (labels) with a trained neural network function, which is both differentiable and able to approximate any mathematical function”.

- The sentence with examples could also be clearer. I feel like the sentence about DFT calculations is a bit too detailed for an overview. Suggestion: “Examples range from classic applications connecting a molecule’s activity with its structure, to more recent networks capable of dramatically accelerating quantum calculations while still retaining accuracy.”

- Second paragraph: “In this book, we will view…” -> “In this book, we will present…”

- Second paragraph: “Much of the data-driven work in the past required decisions about what features are important and create quantitative values that capture them correctly.” is not very clear. “required decisions regarding the choice of features, their importance, and construction of quantitative values that capture them correctly”.

- I feel like the second paragraph doesn’t quite convey why deep learning is better at constructing models that were previously “infeasible or incredibly difficult to construct”.

- Third paragraph: “Another change…”. Change from what? Another recent improvement?

- Third paragraph: again, it’s unclear what model change refers to. A different problem/data/application?

- Third paragraph: the last two sentences seem a bit random in the context, especially if there are students who are not so aware of the deep learning hype. Why would it be a new paradigm of science or a replacement for a chemist? I don’t think this was conveyed in the preceding paragraphs. Also, the last sentence could be much stronger and interesting/inspiring, e.g. something like:
“Although powerful, deep learning is not a new paradigm of science or a replacement for a chemist - it is a tool that is now mature enough to help researchers find the next scientific breakthroughs within molecules and materials.”

**Comment by whitead** [comment]
Hi @cabergh, thanks so much for sharing your feedback and taking the time to write it all out. I will go through these issues and get back to you. 

Since you have read through so much, in your individual issues on each chapter any high-level feedback would be good too. For example, topics in a chapter you think are missed, topics in a chapter that you think were under-explained, etc. I do appreciate all the careful language editing too!
